### ISSUE TITLE
Fonts and spacing in the top entry field

### ISSUE THREAD

**Issue by shvam0000** [yashsehgal/twitter-frontend-clone#12]
![image](https://user-images.githubusercontent.com/60486289/116527883-c70ff680-a8f8-11eb-90b5-cd30ceee90ae.png)

Too many fonts used and font spacing and margins are not appropriate.

**Comment by yashsehgal** [comment]
Yeah, Actually I imported the font family: Poppins but forgot to add it at some places.

**Comment by shvam0000** [comment]
Okay!
